app/templates/events/view/tickets/access/create.hbs

```diff
@@ -0,0 +1,81 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="ui header column">
+      {{t 'Create an access code'}}
+    </div>
+  </div>
+  <div class="row">
+    <div class="sixteen wide column">
+      <form class="ui form">
+        <div class="field">
+          <label class="required">{{t 'ACCESS CODE'}}</label>
+          {{input type='text' name='access_code'}}
+        </div>
+        <div class="field">
+          <label class="required">{{t 'NUMBER OF ACCESS TICKETS'}}</label>
+          {{input type='text' name='number_of_access_tickets'}}
+        </div>
+        <div class="field">
+          <label class="required">{{t 'STATUS'}}</label>
+          <div>
+            {{input type='radio' name='status'}}
+            <label class="required">{{t 'Active'}}</label>
```

Comment:
Doesn't make sense to add the required label here, the one beside Status is enough.